Renaming to baggage (#27)
w3c/baggage: abstract/baggage-overview.md

# Overview

> Baggage represented by set of name/value pairs describing user defined properties of the trace.

Reviewer: ```suggestion
Baggage is a set of name/value pairs describing user defined properties of a distributed request.

```

> `Baggage` header represents user-defined baggage associated with the trace. Libraries and platforms MAY propagate this header.

Reviewer: > user-defined baggage associated with the trace.

redundant

Author: fixed
